PathOfTerraria.csproj

```diff
@@ -27,6 +27,7 @@
     <None Remove="Effects\LunarEffect.glsl" />
     <None Remove="Localization\en-US\Mods.PathOfTerraria.Items.hjson" />
     <None Remove="Localization\en-US\Mods.PathOfTerraria.Projectiles.hjson" />
+    <None Remove="Localization\en-US\Mods.PathOfTerraria.Skills.hjson" />
```

Comment:
Idk where these are being generated from but they prevent rider from searching for the file in the project as it assumes it's a non-project file. Should this be removed?

Reply:
Visual Studio, for some reason, adds these automatically. I don't really know why, and we probably shouldn't be pushing the .csproj to remote anyway. You can remove that, I don't think it'd make any difference (apart from maybe including the file as an embedded resource, which would change the file size barely).